.github/workflows/build-and-test.yaml

```diff
@@ -0,0 +1,89 @@
+# Copyright The OpenTelemetry Authors
+# SPDX-License-Identifier: Apache-2.0
+
+name: build-and-test
+run-name: ${{ github.actor }} is running  GitHub Actions
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    paths:
+      - '.github/workflows/build-and-test.yaml'
+      - ' collector/**'
+      - ' render/**'
+      - ' renderc/**'
+      - ' test/kernel-headers'
+      - ' util/**'
+
+env:
+  PUBLIC_REGISTRY: quay.io/signalfx
+  REGISTRY_IMAGE_PREFIX: splunk-network-explorer-
+  BENV_IMAGE: quay.io/splunko11ytest/network-explorer-debug/benv-final
+  MAVEN_REPOSITORY_URL: https://repo.splunk.com/artifactory/maven
+  GOPROXY: https://repo.splunk.com/artifactory/go
```

Comment:
Are the `repo.splunk.com` repos required?  I believe `repo.splunk.com` is private, so it won't be accessible from public github runners.  Also, we've been advised to remove references to internal splunk domains/urls from public github code.

Reply:
@jeffreyc-splunk , you're spot on, these are not at all required. I will have them removed. These were taken from our Gitlab repo. The evil of copy paste!